src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/tab.scss

```diff
@@ -0,0 +1,63 @@
+@import '../../containers/Application/colors.scss';
+
+$tabTextColor: $black;
+$tabUnderlineBackgroundColor: $black;
+
+.tab {
+    position: relative;
+
+    &:first-child {
+        button {
+            padding-left: 0;
+        }
+
+        &::after {
+            left: 0;
+            width: calc(100% - 20px);
+        }
+    }
+
+    &::after {
+        content: "";
+        width: calc(100% - 40px);
```

Comment:
Maybe introduce a variable for the value `20px`? Then this is maybe less of a question mark 😃